backend/aggregations/app.mjs

```diff
@@ -14,31 +15,35 @@ const floatType = nodeFactory.namedNode(
   "http://www.w3.org/2001/XMLSchema#decimal"
 );
 
+const municipalities = JSON.parse(
+  fs.readFileSync("./data/municipalities.json")
+);
```

Comment:
I think you should be able to just import this file as an ES6 module instead of accessing the filesystem directly :)

```js
import * as municipalities from './data/municipalities.json';
```